README.md

```diff
@@ -16,15 +16,15 @@
 
 <br>
 
-Pixie is an open source observability tool for Kubernetes applications. Use Pixie to view the high-level state of your cluster (service maps, cluster resources, application traffic) and also drill-down into more detailed views (pod state, flame graphs, individual full-body application requests).
+Pixie is an open-source observability tool for Kubernetes applications. Use Pixie to view the high-level state of your cluster (service maps, cluster resources, application traffic) and also drill down into more detailed views (pod state, flame graphs, individual full-body application requests).
```

Comment:
Thanks for the contribution! One quick comment: open source should not be hyphenated as per the [CNCF style guide](https://github.com/cncf/foundation/blob/main/style-guide.md#:~:text=1.-,Cloud%20native%20and%20open%20source,in%20front%20of%20a%20noun).

Reply:
Hi @aimichelle. I'll change it back to 'open source'. Thanks. 

Also, line number 139. Can we extend it to something like this? 
'TCP Drops are pictured graphically between pods and the services/ TCP Drops are pictured in the Live View.' It feels like an incomplete sentence but whatever you say. 

![image](https://user-images.githubusercontent.com/65492460/168810299-d2dbf12f-9319-473b-9a82-5163ccb8a504.png)